app/views/resources/show.html.slim

```diff
@@ -0,0 +1,73 @@
+.page-details
+  .row.page-details__row
+    .col-xs-6.col-md-4
+      h4
+        span class="glyphicon glyphicon--right glyphicon-#{resource_icon(@resource)}"
+        = @resource.resource_type.capitalize
+    
+    .col-xs-6.col-md-8
+      .resource-actions
+        = render 'shared/components/btn_count', path: '#', 
```

Comment:
Changed the columns, now it works except for screens smaller than 290px. If that's not good enough, I will fix the design in another PR. ;)